### PR TITLE
Update example-apps.md

### PR DESCRIPTION
**Pull request by ChrisMM**
* Add header row to table
* Add link for Todo App

@kevgo @mcharawi Thoughts?

### CODE REVIEW

File: example-apps.md [hunk 1]
```diff
@@ -4,14 +4,22 @@ The Exosphere framework provides a number of example applications
 because getting started is easier with working examples.
 
 <table>
+
   <tr>
-    <th>Todo App</th>
+    <th>Example</th>
+    <td>Description</td>
```
Comment (kevgo) [inline example-apps.md]: Let's make the first row bold, not the first column. 
- make line 10 `th`
- make line 14 `td`
- make like 19 `td`

**Comment by ChrisMM** [comment]
Made the first row bold, not the first column.

* made line 10 th
* made line 14 td
* made like 19 td

**Comment by kevgo** [comment]
Looks great! Btw you know you can preview these changes by clicking on the "view" button, right?

**Comment by kevgo** [comment]
I have shipped this. Thanks!

**Comment by ChrisMM** [comment]
re: preview these changes by clicking on the "view" button, right?

Yes - It's a life saver!

**Pull request closed without merging**